contrib/go-pg/pg_go.go

```diff
@@ -0,0 +1,52 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package pg
+
+import (
+	"context"
+
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+
+	"github.com/go-pg/pg/v10"
+)
+
+// Hook wraps the given DB to generate APM data.
+func Hook(db *pg.DB) *pg.DB {
+	db.AddQueryHook(&QueryHook{})
+	return db
+}
+
+// QueryHook for go_pg
+type QueryHook struct{}
+
+// BeforeQuery is executed before query is sent
+// Start measure, when query is started
```

Comment:
Let's remove this documentation (once we make `queryHook` unexported). Same for `AfterQuery`